.github/workflows/sast-scan.yml

```diff
@@ -0,0 +1,11 @@
+name: SAST Scan
+jobs:
+  sast-scan:
```

Comment:
Do we need to set a value for [block_mode](https://github.com/splunk/sast-scanning?tab=readme-ov-file#blockfail-the-sast-job-if-there-are-vulnerabilities-found-)? With our current config, block mode is off. If the intention is to run this with same config as [gitlab ci](https://github.com/signalfx/splunk-otel-collector/blob/main/.gitlab-ci.yml#L91), then it should be policy